src/static/js/index.js

```diff
@@ -40,6 +40,13 @@ if (!Array.prototype.indexOf)
 */
 
 documentReady( function() {
+  buildCharts();
+
+  window.addEventListener( 'resize', buildCharts );
```

Comment:
this resize handler makes the rendering incredibly slow. try throttling it to improve performance when user is resizing window or changing the viewport orientation:

https://developer.mozilla.org/en-US/docs/Web/Events/resize